handbook/community/community.md

```diff
@@ -82,7 +82,7 @@ You will need to install the following tools to use it:
 
 There are several locations in Fleet's public and internal documentation that can be helpful when answering questions raised by the community:
 
-1. Find the frequently asked question (FAQ) documents in each section in the `/docs` folder. These documents are the [Using Fleet FAQ](../docs/Using-Fleet/FAQ.md), [Deploying FAQ](../docs/Deploying/FAQ.md), and [Contributing FAQ](../docs/Contributing/FAQ.md).
+1. Find the frequently asked question (FAQ) documents in each section in the `/docs` folder. These documents are the [Using Fleet FAQ](./../../docs/Using-Fleet/FAQ.md), [Deploying FAQ](./../../docs/Deploying/FAQ.md), and [Contributing FAQ](./../../docs/Contributing/FAQ.md).
```

Comment:
@eashaw Can you tell me more about why you went with this approach instead of flat?

> The easiest way to start is to just keep things flat and use a meta tag to indicate a page is "unlisted" – kinda like how categories work in articles/ today

https://github.com/fleetdm/fleet/issues/6309

Reply:
@mikermcneil I went with the subfolder approach for two reasons:
- To make it easier to tell at a glance what files are included in the main index. Only the files with the same name as their parent folder will be added to the index.
- To make the /handbook folder organized the same way as the /docs folder

Reply:
> Only the files with the same name as their parent folder will be added to the index.

This is how it works in the Sails docs, and I always wished we had used `README.md` instead, which was less confusing and easier to manage when folders get renamed (especially when you're dealing with potentially lots of open PRs in a popular / active project).

That's why in the Fleet `/docs` folder, I set it up to use `README.md` for this purpose instead.

I'm open to exploring changing the plans, but it does take more time, and originally this issue was laid out as:
<img width="1086" alt="image" src="https://user-images.githubusercontent.com/618009/180156469-88477569-337a-4b0e-a2a5-3e65cdc81cad.png">

Reply:
@eashaw Let's proceed with making this work exactly like the docs, contrary to the plan laid out in #6309.

Next steps: https://github.com/fleetdm/fleet/pull/6524#discussion_r926476750